test_rclcpp/test/test_client_scope_consistency_client.cpp

```diff
@@ -39,14 +39,7 @@ TEST(CLASSNAME(service_client, RMW_IMPLEMENTATION), client_scope_consistency_reg
   auto node = rclcpp::Node::make_shared("client_scope_consistency_regression_test");
 
   // Replicate the settings that caused https://github.com/ros2/system_tests/issues/153
-  rmw_qos_profile_t rmw_qos_profile =
-  {
-    RMW_QOS_POLICY_HISTORY_KEEP_LAST,
-    10,
-    RMW_QOS_POLICY_RELIABILITY_RELIABLE,
-    RMW_QOS_POLICY_DURABILITY_VOLATILE,
-    false
-  };
+  rmw_qos_profile_t rmw_qos_profile = rmw_qos_profile_default;
```

Comment:
This doesn't look right, are the settings you're removing the exact same as what's in the default?

Reply:
I double checked this one, and it looks like it is (https://github.com/ros2/rmw/pull/173/files#diff-3b257fc3e609df4f195297e5ee8fe258L43)

Reply:
Ok, never mind, thanks.